voila/handler.py

```diff
@@ -111,8 +111,8 @@ async def get_generator(self, path=None):
                     env_name = f'HTTP_{header_name.upper().replace("-", "_")}'
                     request_info[env_name] = self.request.headers.get(header_name)
 
-        template_arg = self.get_argument("voila-template", None)
-        theme_arg = self.get_argument("voila-theme", None)
+        template_arg = self.get_argument("template", None)
+        theme_arg = self.get_argument("theme", None)
```

Comment:
Should we still support `voila-theme` and `voila-template` to be backward compatible?

Reply:
Let's profit from the freedom of major release 😄

Reply:
and the theme argument is non backward-compatible anyway, users need to use the real theme name, not the package name.

Reply:
Sounds good!